nautobot_firewall_models/homepage.py

```diff
@@ -33,6 +24,30 @@
                 description="Firewall Policies",
                 permissions=["nautobot_firewall_models.view_policyrule"],
             ),
+            HomePageItem(
+                name="NAT Policies",
+                model=NATPolicy,
+                weight=100,
+                link="plugins:nautobot_firewall_models:natpolicy_list",
+                description="NAT Policies",
+                permissions=["nautobot_firewall_models.view_natpolicy"],
+            ),
+            HomePageItem(
+                name="NAT Policy Rules",
+                model=NATPolicyRule,
+                weight=200,
```

Comment:
Can you make sure each weight is unique to enforce intended ordering?

Reply:
Addressed.